app/core/tokenList.json

```diff
@@ -428,6 +428,19 @@
       }
     }
   },
+  "NNN": {
+    "symbol": "NNN",
+    "companyName": "Novem Gold AG",
+    "type": "NEP5",
+    "networks": {
+      "1": {
+        "name": "Novem Gold Token",
+        "hash": "50091057ff12863f1a43266b9786209e399c6ffc",
+        "decimals": 8,
+        "totalSupply": 0
```

Comment:
update supply

Reply:
updated it to 1 and did not resolve my issue.